chapters/data/main.tex

```diff
@@ -604,292 +597,275 @@ \subsection{Pairwise correlation}\label{subsec:diab_correlation}
+well-being of a patient~\cite{Deschenes2015,Klimek2015,Walker2016}.
+
+As in the previous section, the bubble plot in Figure~\ref{fig:diab_bubble}
+allows these quantities to be considered simultaneously. Again, and despite the
+efficacy of the visualisation itself, there is little insight to be gained.
+There are no distinctly important components here and the system seems to be
+optimised for both the diabetic and non-diabetic populations. This
+`optimisation' is only up to the point where the smallest relative variation of
+a component is still twice its mean.
 
 
 \subsection{Resource consumption}\label{subsec:diab_resources}
 
 The types of comparisons made between the non-diabetic and diabetic populations
 throughout this analysis are useful for observing their similarities in a direct
-way, and in understanding how the groups may relate to one another.
-
-However, these are not the only devices available for examining such a subset of
-the data. Particularly when looking at costing data such as this, another useful
-way of evaluating a subset is to quantify its size and representation in the
-data with respect to various cost-indicative attributes. These attributes can
-give a sense of the level and nature of the resources that are consumed by the
-population in question. Namely, these attributes are: the proportion of total
-net costs and admissions, and length of stay. During this part of the analysis,
-these attributes will be referred to as the ``chosen'' attributes.
-
-In addition, whilst considering that costs are the focus of this body of
-chapter, it can be useful to investigate how certain cost-related quantities
-evolve for a subset of the population. In this section of the analysis, the
-evolution of the aforementioned attributes will be discussed within the diabetic
-population as a part of the general data population. For these purposes, the
-data must be manipulated into a chronological form and so some approximations
-have to be made. Here, each of the chosen attributes is given with respect to a
-particular admission date, and has been calculated in the following way for each
-admission date:
-
+way, and in understanding how the groups may relate to one another. However,
+these are not the only devices available for examining such a subset of the
+data. Particularly when looking at costing data such as this, another useful way
+of evaluating a subset is to quantify its contributions to those costs over time
+with regard to the general population. The attributes used to identify these
+contributions can give a sense of the level and nature of the resources that are
+consumed by the population in question. This stage of the analysis considers the
+following attributes: the proportion of net costs and admissions, and the length
+of stay. 
+
+For these purposes, the data must be manipulated into a chronological form.
+Therefore, some approximations must be made. Here, each of the chosen attributes
```

Comment:
Are these approximations or just the specific way we're choosing to define these things?

Reply:
This is true. The explanation after this (lines 776-783) covers the issues that I have called "approximations" here. I'll get rid of that phrase now.

```suggestion
Here, each of the chosen attributes
```